capture/index.js

```diff
@@ -92,8 +92,10 @@ app.get('/*', async (req, res) => {
     console.timeLog(timeLabel, 'page loaded')
 
     if (commentId) {
-      console.timeLog(timeLabel, 'scrolling to comment')
       await page.waitForSelector('.outline-it')
+      setTimeout(() => {
+        console.timeLog(timeLabel, 'scrolling to comment')
+      }, 200)
```

Comment:
`setTimeout` runs async so this will return immediately then proceed to screenshot.

What you want to do here is something like `await new Promise((resolve, reject) => setTimeout(resolve, 100))` to block the function execution.

I think 100ms should be plenty of time too btw.

Reply:
Ok! thanks

Reply:
caught a lightning bolt in this one

![image](https://github.com/stackernews/stacker.news/assets/108441023/9cebe08b-ec4a-410d-9222-dea9e6fd0072)